srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Manager/Soap/MailService.php

```diff
@@ -13,22 +13,22 @@ public function sendItem($playerName, $subject, $message, $itemId, $stack) {
         $_subject = addslashes($subject);
         $_itemId = intval($itemId);
         $_stack = intval($stack);
-        return $this->executeCommand('.send items  ' . $playerName . ' "' . $_subject . '" "' . $_message . '" ' . $_itemId . ':' . $_stack);
+        return $this->executeCommand(".send items $playerName \"$_subject\" \"$_message\" $_itemId :$_stack");
```

Comment:
the issue is here, I don't really know why, but if you revert this like it was before, it works

Reply:
(I don't see still any log in the soap log)

Reply:
I think I found why you got an empty email, I add an space between the itemId and the : before the stack, `$_itemId :$_stack"`.

I will take a look why the log is not saved.